guides/user/pipeline-expressions/index.md

```diff
@@ -80,6 +71,18 @@ You can add expressions to be checked here by selecting *Expression* from the Ch
 
 When the pipeline runs, if this expression evaluates to true, the pipeline will continue as is. If not, the stages following this stage will not run. 
 
+### Sometimes you can't use an expression
```

Comment:
I'd add a newline after this header. Some markdown rendering requires it.

Reply:
Will do